### PR TITLE
[5.1] Allow config['schema'] to be array for PostgresConnector

### PR DESCRIPTION
**Pull request by FractalizeR**
Postgres allows several schemas in search_path. This PR will partially solve #8523. This is backwards compatible with what we have now.

http://www.postgresql.org/docs/9.4/static/ddl-schemas.html#DDL-SCHEMAS-PATH

### CODE REVIEW

File: tests/Database/DatabaseConnectorTest.php [hunk 1]
```diff
@@ -73,6 +73,24 @@ public function testPostgresSearchPathIsSet()
         $this->assertSame($result, $connection);
     }
 
+
```
Comment (GrahamCampbell) [inline tests/Database/DatabaseConnectorTest.php]: too many newlines

File: tests/Database/DatabaseConnectorTest.php [hunk 2]
```diff
@@ -73,6 +73,24 @@ public function testPostgresSearchPathIsSet()
...
+		$connector->expects($this->once())->method('getOptions')->with($this->equalTo($config))->will($this->returnValue(['options']));
+		$connector->expects($this->once())->method('createConnection')->with($this->equalTo($dsn), $this->equalTo($config), $this->equalTo(['options']))->will($this->returnValue($connection));
+		$connection->shouldReceive('prepare')->once()->with('set names \'utf8\'')->andReturn($connection);
+		$connection->shouldReceive('prepare')->once()->with('set search_path to "public", "user"')->andReturn($connection);
+		$connection->shouldReceive('execute')->twice();
+		$result = $connector->connect($config);
+
+		$this->assertSame($result, $connection);
+	}
+
```
Comment (GrahamCampbell) [inline tests/Database/DatabaseConnectorTest.php]: too many newlines

**Comment by FractalizeR** [comment]
Done. Is there any tool for checking Laravel code style? :)

**Comment by GrahamCampbell** [comment]
> Done. Is there any tool for checking Laravel code style? :)

Yes php-cs-fixer. We have a config file in this repo for it.

**Comment by GrahamCampbell** [comment]
It's still missing support for some of our phpdoc rules, but it does most of our rules. :)

**Comment by FractalizeR** [comment]
Oh, that's great! I will get it for my laravel fork.

**Pull request merged**